common/src/main/java/com/datastrato/gravitino/dto/rel/DistributionDTO.java

```diff
@@ -22,14 +23,16 @@
 public class DistributionDTO implements Distribution {
 
   public static final DistributionDTO NONE =
-      new Builder().withStrategy(Strategy.HASH).withNumber(0).withArgs(EMPTY_ARGS).build();
+      new Builder().withStrategy(Strategy.NONE).withNumber(0).withArgs(EMPTY_ARGS).build();
 
   // Distribution strategy/method
   private final Strategy strategy;
 
+  @JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
@yuqi1129 are you fine with this change?

Reply:
Since this value is meaningless when using `Strategy.NONE`, it's okay for me to make it optional here. 
But I suggest we need to add some checks for this value when the strategy is not `NONE`, such as when using `EVEN`, the value should be great than 0.